lib/Plack/Request.pm

```diff
@@ -485,11 +485,20 @@ Shortcut to $req->headers->content_encoding.
 
 =item content_length
 
-Shortcut to $req->headers->content_length.
+Returns the raw value of the Content-Length header.
+
+Before version 0.9925, this method was a shortcut for
+C<$req->headers->content_length>.
```

Comment:
i think you want `C<< $req->headers->content_length >>` ?

Reply:
Ah, yes, of course.  Repushed just now.